0-SCore/Scripts/UtilityAI/UAITaskAttackTargetEntitySDX.cs

```diff
@@ -149,28 +149,28 @@ public override void Update(Context _context)
                 return;
             }
 
-            var num = UAIUtils.DistanceSqr(_context.Self.position, position);
+            //var num = UAIUtils.DistanceSqr(_context.Self.position, position);
```

Comment:
Some of the code that I commented out here, is because it is unused, and I wanted to save the CPU cycles. But I did not want to remove it outright in case you needed to restore it later.